components/string.rst

```diff
@@ -552,6 +552,17 @@ from GitHub or Slack, use the first argument of ``withEmoji()`` method::
     $slug = $slugger->slug('a 😺, 🐈‍⬛, and a 🦁');
     // $slug = 'a-smiley-cat-black-cat-and-a-lion';
 
+If you want to strip emojis from a string, you can also use the special
+``strip`` locale::
+
+    use Symfony\Component\String\Slugger\AsciiSlugger;
+
+    $slugger = new AsciiSlugger();
+    $slugger = $slugger->withEmoji('strip');
+
+    $slug = $slugger->slug('a 😺, 🐈‍⬛, and a 🦁');
+    // $slug = 'a-and-a';
```

Comment:
versionadded missing, or did you add one while merging @javiereguiluz ?

Cannot check I am currently on a phone 📱

Reply:
He added it in https://github.com/symfony/symfony-docs/commit/874f0260708961aad282888ce2ade08e5a6ccada, thanks for this Javier!

Reply:
Thanks for the feedback